.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,7 @@
 ## Tasks
 
+<!-- Sorry, we're only accepting minor bugfix PRs from the community, no big change or new feature -->
```

Comment:
Hi @dreamwhisper,
What do you think about this?

Small bugfix PRs would help, but it tells people to not open a big PR that we won't review.

Reply:
This wording changed, so resolving this conversation